7.8.Вложенные_циклы._Часть_1/7.8.5.py

```diff
@@ -0,0 +1,5 @@
+n = int(input())
+for i in range(n):
+    for i in range(2):         
```

Comment:
1) Нельзя называть переменные вложенного цикла одинакого.
2) Так как они у тебя не используются, называй их `_`.
```suggestion
for _ in range(n):
    for _ in range(2):         
```